invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/AccessRequests/AccessRequestsTab.js

```diff
@@ -72,19 +72,22 @@ export class AccessRequestsTab extends Component {
   };
 
   initFormValues = () => {
-    const { record } = this.props;
+    const { record, isAccessLinksExpirationRequired } = this.props;
 
     const settings = record.parent.access.settings;
 
     if (!_has(settings, "secret_link_expiration")) {
       settings["secret_link_expiration"] = 0;
     }
+    else if (settings["secret_link_expiration"] === 0 && isAccessLinksExpirationRequired) {
```

Comment:
```suggestion
} else if (settings["secret_link_expiration"] === 0 && isAccessLinksExpirationRequired) {
```
looks like the linter wants it to be on the same line